lte/gateway/c/sctpd/sctpd_downlink_impl.h

```diff
@@ -40,17 +43,18 @@ class SctpdDownlinkImpl final : public SctpdDownlink::Service {
     override;
 
   // Implementation of SctpdDownlink.SendDl method (see sctpd.proto for more info)
-  Status SendDl(
-    ServerContext *context,
-    const SendDlReq *request,
-    SendDlRes *response) override;
+  Status SendDl( ServerContext *context, const SendDlReq *request, SendDlRes *response)
+    override;
+  Status create_sctp_connection(std::unique_ptr<SctpConnection>& sctp_connection,
```

Comment:
Add a comment describing the function and please add new line before the comment

Reply:
okay